pkg/licensing/normalize_test.go

```diff
@@ -8,6 +8,219 @@ import (
 	"github.com/aquasecurity/trivy/pkg/licensing"
 )
 
+func TestMap(t *testing.T) {
+	assert.Empty(t, licensing.InvalidMappingKeys((nil)))
```

Comment:
```suggestion
	assert.Empty(t, licensing.InvalidMappingKeys(nil))
```

Reply:
InvalidMappingKeys no more used in latest commit.